.github/workflows/actions.yml

```diff
@@ -21,9 +21,7 @@ jobs:
       - name: install go
         uses: actions/setup-go@v2
         with:
-          go-version: 1.12
-      - name: install
-        run: make install
+          go-version: "1.20"
```

Comment:
is the double quote necessary?

Reply:
Yes—without it, YAML interprets this as "1.2" and you get a very old version of Go.